### PR TITLE
item_db and item_combo_db correction #9

### PR DESCRIPTION
**Pull request by kaninhot004**
* **Server Mode**: 

Renewal

* **Description of Pull Request**: 

item_db and item_combo_db update from reading Divine Pride (kRO server selected).

https://www.divine-pride.net/database/item/5920
https://www.divine-pride.net/database/item/2166
https://www.divine-pride.net/database/item/1691
https://www.divine-pride.net/database/item/19021
https://www.divine-pride.net/database/item/22106
https://www.divine-pride.net/database/item/22106
https://www.divine-pride.net/database/item/15169
https://www.divine-pride.net/database/item/15169
https://www.divine-pride.net/database/item/5966
https://www.divine-pride.net/database/item/5966
https://www.divine-pride.net/database/item/5966
https://www.divine-pride.net/database/item/13300
https://www.divine-pride.net/database/item/13301
https://www.divine-pride.net/database/item/13302
https://www.divine-pride.net/database/item/13303
https://www.divine-pride.net/database/item/13304
https://www.divine-pride.net/database/item/13307
https://www.divine-pride.net/database/item/13308
https://www.divine-pride.net/database/item/13309
https://www.divine-pride.net/database/item/13315
https://www.divine-pride.net/database/item/13316
https://www.divine-pride.net/database/item/13317
https://www.divine-pride.net/database/item/13322
https://www.divine-pride.net/database/item/13323
https://www.divine-pride.net/database/item/13327
https://www.divine-pride.net/database/item/13328
https://www.divine-pride.net/database/item/13332
https://www.divine-pride.net/database/item/13403
https://www.divine-pride.net/database/item/13411
https://www.divine-pride.net/database/item/13412
https://www.divine-pride.net/database/item/13413
https://www.divine-pride.net/database/item/13417
https://www.divine-pride.net/database/item/13418
https://www.divine-pride.net/database/item/13419
https://www.divine-pride.net/database/item/100273
https://www.divine-pride.net/database/item/100274
https://www.divine-pride.net/database/item/100275
https://www.divine-pride.net/database/item/100276
https://www.divine-pride.net/database/item/100277
https://www.divine-pride.net/database/item/100327
https://www.divine-pride.net/database/item/100328
https://www.divine-pride.net/database/item/100329
https://www.divine-pride.net/database/item/100330
https://www.divine-pride.net/database/item/100331
https://www.divine-pride.net/database/item/100360
https://www.divine-pride.net/database/item/100361
https://www.divine-pride.net/database/item/100362
https://www.divine-pride.net/database/item/100363
https://www.divine-pride.net/database/item/100376
https://www.divine-pride.net/database/item/100377
https://www.divine-pride.net/database/item/100378
https://www.divine-pride.net/database/item/100379
https://www.divine-pride.net/database/item/100388
https://www.divine-pride.net/database/item/100390
https://www.divine-pride.net/database/item/100393
https://www.divine-pride.net/database/item/100394
https://www.divine-pride.net/database/item/100395
https://www.divine-pride.net/database/item/100396
https://www.divine-pride.net/database/item/100397
https://www.divine-pride.net/database/item/100398
https://www.divine-pride.net/database/item/100399
https://www.divine-pride.net/database/item/100400
https://www.divine-pride.net/database/item/100401
https://www.divine-pride.net/database/item/100437
https://www.divine-pride.net/database/item/22238

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -13855,7 +13890,51 @@
...
+640012,Up_Blue_Crystal_Staff,Patent Blue Crystal Staff,5,,,1000,100:235,,1,2,0x00000200,56,2,2,4,150,1,10,{ bonus bUnbreakableWeapon; .@r = getrefine(); bonus bMatk,.@r/2*15; bonus2 bSkillAtk,"WL_JACKFROST",.@r/3*12; if (.@r>=7){ bonus bVariableCastrate,-10; bonus2 bMagicAddEle,Ele_Water,15; bonus2 bMagicAddEle,Ele_Wind,15; } if (.@r>=9){ bonus2 bSkillCooldown,"WL_JACKFROST",-1000; bonus bDelayRate,-5; } if (.@r>=11){ bonus2 bMagicAddSize,Size_All,15; bonus2 bSkillAtk,"WL_CHAINLIGHTNING",15; } },{},{}
+640004,Blue_Crystal_Staff,Blue Crystal Staff,5,,,800,100:200,,1,2,0x00000200,56,2,2,4,100,1,10,{ bonus bUnbreakableWeapon; .@r = getrefine(); bonus bMatk,.@r/2*10; bonus2 bSkillAtk,"WL_JACKFROST",.@r/3*12; if (.@r>=9) { bonus2 bAddEle,Ele_Water,7; } if (.@r>=11) { bonus2 bSkillCooldown,"WL_JACKFROST",-1000; bonus2 bAddEle,Ele_Fire,7; } },{},{}
```
Comment (crazy-arashi) [inline db/re/item_db.txt:13924]:
```suggestion
640004,Blue_Crystal_Staff,Blue Crystal Staff,5,,,800,100:200,,1,2,0x00000200,56,2,2,4,100,1,10,{ bonus bUnbreakableWeapon; .@r = getrefine(); bonus bMatk,.@r/2*10; bonus2 bSkillAtk,"WL_JACKFROST",.@r/3*12; if (.@r>=9) { bonus2 bMagicAddEle,Ele_Water,7; } if (.@r>=11) { bonus2 bSkillCooldown,"WL_JACKFROST",-1000; bonus2 bMagicAddEle,Ele_Fire,7; } },{},{}
```
Also, arrange the order of ID

File: db/re/item_db.txt [hunk 2]
```diff
@@ -13855,7 +13890,51 @@
...
+650004,Up_Humma_Clear,Patent Huuma Shuriken Clearness,5,20,,1300,230,,1,2,0x22000000,63,2,34,4,150,1,22,{ bonus bUnbreakableWeapon; .@r = getrefine(); bonus bBaseAtk,15*(.@r/2); bonus bLongAtkRate,.@r/3*2; if (.@r>=7){ bonus2 bSkillAtk,"KO_HUUMARANKA",30; bonus bVariableCastrate,-10; } if (.@r>=9){ bonus2 bSkillCooldown,"KO_HUUMARANKA",-1000; } if (.@r>=11){ bonus2 bAddEle,Ele_Fire,20; bonus2 bAddEle,Ele_Dark,20; bonus2 bAddEle,Ele_Holy,20; bonus2 bAddRace,RC_Undead,20; bonus2 bAddRace,RC_Demon,20; } },{},{}
```
Comment (crazy-arashi) [inline db/re/item_db.txt:13925]:
```suggestion
650004,Up_Humma_Clear,Patent Huuma Shuriken Clearness,5,20,,2300,290,,1,2,0x22000000,63,2,34,4,150,1,22,{ bonus bUnbreakableWeapon; .@r = getrefine(); bonus bBaseAtk,15*(.@r/2); bonus bLongAtkRate,.@r/3*2; if (.@r>=7){ bonus2 bSkillAtk,"KO_HUUMARANKA",30; bonus bVariableCastrate,-10; } if (.@r>=9){ bonus2 bSkillCooldown,"KO_HUUMARANKA",-1000; } if (.@r>=11){ bonus2 bAddEle,Ele_Fire,20; bonus2 bAddEle,Ele_Dark,20; bonus2 bAddEle,Ele_Holy,20; bonus2 bAddRace,RC_Undead,20; bonus2 bAddRace,RC_Demon,20; bonus2 bAddRace,RC_Angel,20; } },{},{}
```

File: db/re/item_db.txt [hunk 3]
```diff
@@ -13855,7 +13890,51 @@
...
+
+700003,Scalet_Dragon_L_Bow,Scarlet Dragon Leather Bow,5,,,800,170,,5,2,0x00000800,56,2,34,4,100,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*10; bonus bDelayRate,.@r/3*4; if (.@r>=9) { bonus2 bSkillAtk,"RA_AIMEDBOLT",35; } if (.@r>=11) { bonus2 bSkillCooldown,"RA_AIMEDBOLT",-1000; } },{},{}
+700013,Up_Narcis_Bow,Patent Narcissus Bow,5,0,,1000,210,,5,2,0x00080800,58,2,34,4,150,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*15; if (.@r>=7){ bonus2 bSkillCooldown,"RA_ARROWSTORM",-2000; } if (.@r>=9){ bonus2 bSkillAtk,"RA_ARROWSTORM",15; } if (.@r>=11){ bonus2 bSkillAtk,"RA_ARROWSTORM",10; bonus bLongAtkRate,15; } },{},{}
```
Comment (crazy-arashi) [inline db/re/item_db.txt:13928]:
```suggestion
700013,Up_Narcis_Bow,Patent Narcissus Bow,5,,,1000,210,,5,2,0x000A0800,56,2,34,4,150,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*15; bonus bLongAtkRate,.@r/3*4; if (.@r>=7){ bonus2 bSkillCooldown,"WM_SEVERE_RAINSTORM",-2000; } if (.@r>=9){ bonus2 bSkillAtk,"WM_SEVERE_RAINSTORM",15; } if (.@r>=11){ bonus2 bSkillAtk,"WM_SEVERE_RAINSTORM",10; bonus2 bWeaponDamageRate,W_BOW,15; } },{},{}
```

File: db/re/item_db.txt [hunk 4]
```diff
@@ -13855,7 +13890,51 @@
...
+700018,Up_Royal_Bow_K,Patent Royal Bow,5,,,1200,205,,5,2,0x000A0808,63,2,34,3,150,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*15; bonus2 bSkillAtk,"RA_ARROWSTORM",13*(.@r/3); if (.@r>=7){ bonus bAspdRate,10; bonus bLongAtkRate,15; } if (.@r>=9){ bonus bLongAtkRate,12; bonus bCritAtkRate,10; } if (.@r>=11){ bonus2 bSkillAtk,"SN_SHARPSHOOTING",20; bonus bDelayRate,-10; } },{},{}
```
Comment (crazy-arashi) [inline db/re/item_db.txt:13929]:
```suggestion
700018,Up_Royal_Bow_K,Patent Royal Bow,5,,,1200,205,,5,2,0x000A0800,56,2,34,4,150,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*15; bonus2 bSkillAtk,"RA_ARROWSTORM",12*(.@r/3); if (.@r>=7){ bonus bAspdRate,10; bonus2 bWeaponDamageRate,W_BOW,15; } if (.@r>=9){ bonus bLongAtkRate,12; bonus bCritAtkRate,10; } if (.@r>=11){ bonus2 bSkillAtk,"SN_SHARPSHOOTING",20; bonus bDelayRate,-10; } },{},{}
```

File: db/re/item_db.txt [hunk 5]
```diff
@@ -13855,7 +13890,51 @@
...
+640004,Blue_Crystal_Staff,Blue Crystal Staff,5,,,800,100:200,,1,2,0x00000200,56,2,2,4,100,1,10,{ bonus bUnbreakableWeapon; .@r = getrefine(); bonus bMatk,.@r/2*10; bonus2 bSkillAtk,"WL_JACKFROST",.@r/3*12; if (.@r>=9) { bonus2 bAddEle,Ele_Water,7; } if (.@r>=11) { bonus2 bSkillCooldown,"WL_JACKFROST",-1000; bonus2 bAddEle,Ele_Fire,7; } },{},{}
+650004,Up_Humma_Clear,Patent Huuma Shuriken Clearness,5,20,,1300,230,,1,2,0x22000000,63,2,34,4,150,1,22,{ bonus bUnbreakableWeapon; .@r = getrefine(); bonus bBaseAtk,15*(.@r/2); bonus bLongAtkRate,.@r/3*2; if (.@r>=7){ bonus2 bSkillAtk,"KO_HUUMARANKA",30; bonus bVariableCastrate,-10; } if (.@r>=9){ bonus2 bSkillCooldown,"KO_HUUMARANKA",-1000; } if (.@r>=11){ bonus2 bAddEle,Ele_Fire,20; bonus2 bAddEle,Ele_Dark,20; bonus2 bAddEle,Ele_Holy,20; bonus2 bAddRace,RC_Undead,20; bonus2 bAddRace,RC_Demon,20; } },{},{}
+
+700003,Scalet_Dragon_L_Bow,Scarlet Dragon Leather Bow,5,,,800,170,,5,2,0x00000800,56,2,34,4,100,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*10; bonus bDelayRate,.@r/3*4; if (.@r>=9) { bonus2 bSkillAtk,"RA_AIMEDBOLT",35; } if (.@r>=11) { bonus2 bSkillCooldown,"RA_AIMEDBOLT",-1000; } },{},{}
```
Comment (crazy-arashi) [inline db/re/item_db.txt:13927]:
```suggestion
700003,Scalet_Dragon_L_Bow,Scarlet Dragon Leather Bow,5,,,800,170,,5,2,0x00000800,56,2,34,4,100,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*10; bonus bDelayRate,-(.@r/3*4); if (.@r>=9) { bonus2 bSkillAtk,"RA_AIMEDBOLT",35; } if (.@r>=11) { bonus2 bSkillCooldown,"RA_AIMEDBOLT",-1000; } },{},{}
```

File: db/re/item_db.txt [hunk 6]
```diff
@@ -13855,7 +13890,51 @@
...
+700013,Up_Narcis_Bow,Patent Narcissus Bow,5,0,,1000,210,,5,2,0x00080800,58,2,34,4,150,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*15; if (.@r>=7){ bonus2 bSkillCooldown,"RA_ARROWSTORM",-2000; } if (.@r>=9){ bonus2 bSkillAtk,"RA_ARROWSTORM",15; } if (.@r>=11){ bonus2 bSkillAtk,"RA_ARROWSTORM",10; bonus bLongAtkRate,15; } },{},{}
+700018,Up_Royal_Bow_K,Patent Royal Bow,5,,,1200,205,,5,2,0x000A0808,63,2,34,3,150,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*15; bonus2 bSkillAtk,"RA_ARROWSTORM",13*(.@r/3); if (.@r>=7){ bonus bAspdRate,10; bonus bLongAtkRate,15; } if (.@r>=9){ bonus bLongAtkRate,12; bonus bCritAtkRate,10; } if (.@r>=11){ bonus2 bSkillAtk,"SN_SHARPSHOOTING",20; bonus bDelayRate,-10; } },{},{}
+700019,Up_Scalet_Dragon_L_Bow,Patent Scarlet Dragon Leather Bow,5,,,900,200,,5,2,0x00000800,56,2,34,4,150,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*15; bonus bDelayRate,.@r/3*4; if (.@r>=7){ bonus bLongAtkRate,10; bonus2 bAddSize,Size_All,10; } if (.@r>=9){ bonus2 bSkillAtk,"RA_AIMEDBOLT",35; } if (.@r>=11){ bonus2 bSkillCooldown,"RA_AIMEDBOLT",-1000; bonus bVariableCastrate,-10; } },{},{}
```
Comment (crazy-arashi) [inline db/re/item_db.txt:13930]:
```suggestion
700019,Up_Scalet_Dragon_L_Bow,Patent Scarlet Dragon Leather Bow,5,,,900,200,,5,2,0x00000800,56,2,34,4,150,1,11,{ .@r = getrefine(); bonus bBaseAtk,.@r/2*15; bonus bDelayRate,-(.@r/3*4); if (.@r>=7){ bonus bLongAtkRate,10; bonus2 bAddSize,Size_All,10; } if (.@r>=9){ bonus2 bSkillAtk,"RA_AIMEDBOLT",35; } if (.@r>=11){ bonus2 bSkillCooldown,"RA_AIMEDBOLT",-1000; bonus bVariableCastrate,-10; } },{},{}
```

File: db/re/item_db.txt [hunk 7]
```diff
@@ -13855,7 +13890,51 @@
...
+810001,Up_Master_Soul_Rifle,Patent Master Soul Rifle,5,20,,1000,200:250,,9,2,0x40000000,63,2,34,4,150,1,18,{ .@r = getrefine(); bonus bMatk,15*(.@r/3); bonus bAspdRate,(.@r/3*3); if (.@r>=7){ bonus2 bSkillAtk,"WL_SOULEXPANSION",20; bonus2 bMagicAddEle,Ele_Ghost,15; } if (.@r>=9){ bonus5 bAutoSpell,"MG_SOULSTRIKE",10,15,BF_LONG,1; } if (.@r>=11){ bonus5 bAutoSpell,"HW_NAPALMVULCAN",3,7,BF_LONG,1; bonus5 bAutoSpell,"WL_SOULEXPANSION",3,7,BF_LONG,1; } },{},{}
+820001,Up_Demon_Slayer_Shot,Patent Demon Slayer Shot,5,,,1400,275,,9,2,0x40000000,63,2,34,4,150,1,20,{ .@r = getrefine(); bonus bSplashRange,1; bonus bLongAtkRate,3*(.@r/2); bonus bMaxSPrate,2*(.@r/3); if (.@r>=7){ bonus2 bAddClass,Class_Boss,15; bonus2 bAddRace,RC_Undead,15; bonus2 bAddRace,RC_Demon,15; } if (.@r>=9){ autobonus3 "{ bonus bBaseAtk,50; bonus bLongAtkRate,10; }",1000,90000,"RL_P_ALTER"; } if (.@r>=11){ bonus2 bSkillAtk,"RL_BANISHING_BUSTER",35; bonus2 bSkillAtk,"RL_S_STORM",35; } },{},{}
```
Comment (crazy-arashi) [inline db/re/item_db.txt:13934]:
```suggestion
820001,Up_Demon_Slayer_Shot,Patent Demon Slayer Shot,5,,,2300,295,,9,2,0x40000000,63,2,34,4,150,1,20,{ .@r = getrefine(); bonus bSplashRange,1; bonus bLongAtkRate,3*(.@r/2); bonus bMaxSPrate,2*(.@r/3); if (.@r>=7){ bonus2 bAddClass,Class_Boss,15; bonus2 bAddRace,RC_Undead,15; bonus2 bAddRace,RC_Demon,15; } if (.@r>=9){ autobonus3 "{ bonus bBaseAtk,50; bonus bLongAtkRate,10; }",1000,90000,"RL_P_ALTER"; } if (.@r>=11){ bonus2 bSkillAtk,"RL_BANISHING_BUSTER",35; bonus2 bSkillAtk,"RL_S_STORM",35; } },{},{}
```

File: db/re/item_db.txt [hunk 8]
```diff
@@ -13855,7 +13890,51 @@
...
+830002,Up_The_Black,Patent The Black,5,,,3000,225,,9,2,0x40000000,63,2,34,4,150,1,19,{ .@r = getrefine(); bonus bBaseAtk,15*(.@r/2); bonus bLongAtkRate,4*(.@r/3); if (.@r>=7){ bonus2 bSkillAtk,"RL_R_TRIP",20; bonus2 bSkillUseSPrate,"RL_R_TRIP",-10; } if (.@r>=9){ bonus2 bSkillAtk,"RL_R_TRIP",15; autobonus3 "{ bonus bBaseAtk,50; bonus bLongAtkRate,10; }",1000,90000,"RL_P_ALTER"; } if (.@r>=11){ bonus2 bAddEle,Ele_All,15; } },{},{}
```
Comment (crazy-arashi) [inline db/re/item_db.txt:13935]:
```suggestion
830002,Up_The_Black,Patent The Black,5,,,3000,225,,9,2,0x40000000,63,2,34,4,150,1,19,{ .@r = getrefine(); bonus bBaseAtk,15*(.@r/2); bonus bLongAtkRate,4*(.@r/3); if (.@r>=7){ bonus2 bSkillAtk,"RL_R_TRIP",20; bonus2 bSkillUseSPrate,"RL_R_TRIP",10; } if (.@r>=9){ bonus2 bSkillAtk,"RL_R_TRIP",15; autobonus3 "{ bonus bBaseAtk,50; bonus bLongAtkRate,10; }",1000,90000,"RL_P_ALTER"; } if (.@r>=11){ bonus2 bAddSize,Size_All,15; } },{},{}
```

**Review by crazy-arashi — CHANGES_REQUESTED** [review]

**Comment by kaninhot004** [comment]
Thank you @crazy-arashi !

All changes is done.

**Review by crazy-arashi — APPROVED** [review]

**Comment by Atemo** [comment]
Do you mind to update the item sql too in your PRs? (uses the tool https://github.com/rathena/rathena/blob/master/tools/convert_sql.pl)

**Comment by kaninhot004** [comment]
> Do you mind to update the item sql too in your PRs? (uses the tool https://github.com/rathena/rathena/blob/master/tools/convert_sql.pl)

Yep, I will try that.

**Review by Atemo — CHANGES_REQUESTED** [review]
GW !

**Comment by kaninhot004** [comment]
All changes has been fixed. Thank you @Atemo

**Review by Atemo — APPROVED** [review]

File: db/re/item_combo_db.txt [hunk 9]
```diff
@@ -1438,3 +1442,49 @@
...
+//22238:550014,{ bonus bLuk,10; bonus2 bAddClass,Class_All,7; bonus bMatkRate,7; } /*Not had item in item_db yet*/
+//22238:510022:28763,{ bonus bDex,10; bonus2 bAddClass,Class_All,7; } /*Not had item in item_db yet*/
+//22238:650004,{ bonus bStr,10; bonus bLongAtkRate,5; } /*Not had item in item_db yet*/
+//22238:540010,{ bonus bStr,10; bonus2 bAddClass,Class_All,7; } /*Not had item in item_db yet*/
+//22238:550015,{ bonus bInt,10; bonus bMatkRate,7; } /*Not had item in item_db yet*/
+//22238:800002,{ bonus bDex,10; bonus2 bAddClass,Class_All,7; } /*Not had item in item_db yet*/
+//22238:810001,{ bonus bInt,10; bonus2 bMagicAddEle,Ele_Ghost,5; } /*Not had item in item_db yet*/
+//22238:840001,{ bonus bDex,10; bonus bLongAtkRate,5; } /*Not had item in item_db yet*/
+//22238:830002,{ bonus bDex,10; bonus bLongAtkRate,5; } /*Not had item in item_db yet*/
+//22238:820001,{ bonus bDex,10; bonus bLongAtkRate,5; } /*Not had item in item_db yet*/
```
Comment (crazy-arashi) [inline db/re/item_combo_db.txt:1489]: Please remove the comments. 22238 is in the database already.

**Review by crazy-arashi — CHANGES_REQUESTED** [review]

**Review by crazy-arashi — APPROVED** [review]

**Review by aleos89 — APPROVED** [review]

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**